config/puma.rb

```diff
@@ -15,6 +15,6 @@
   end
 end
 
-port = Integer(ENV['PORT'] || 3000)
+port = 5000
```

Comment:
Without `foreman` (which we believe was setting `ENV['PORT']` to 5000), let's just hard-code the port to be 5000

Reply:
Given that some macbooks appear to use 5000 for airplay, would it be okay if we used another port like 3000 or 8000?

Reply:
+1 to what @aaronbrezel says - I don't think this gets around the root issue, which is that port 5000 is already in use by the Mac OS. 

I also think it's still worth leaving an override to use if necessary, rather than hard-coding a port, in case you need to run it on another port for whatever reason. 3000 and 8000 are frequently used ports and you might want to set it locally to something slightly different like 3010 or 8010, for example

Reply:
One other thing I noticed here is that the file is looking for `ENV['PORT']`, which isn't in our env files. My guess is this is probably why the port wasn't being updated, and that it didn't have anything to do with `foreman`. 

**Update** - just confirmed that setting a `PORT` value in your env file, `PORT=4000` for example, works